wiki/Layout-component/universal-banner.md

```diff
@@ -0,0 +1,146 @@
+
+###### Logic
+
+|         | condition            | description                                                                      |
+| :------ | :------------------- | :------------------------------------------------------------------------------- |
+| `1.0.1` | `@sm` `@md`          | Media is hidden                                                                  |
+| `1.0.2` | `onClick` && `video` | If the user clicks a video, than play the video in the [[Lightbox media viewer]] |
+
+###### Sizes
+
+|         | name | description |           |
+| :------ | :--- | :---------- | :-------- |
+| `1.1.1` | `md` | 4 columns   | `default` |
+| `1.1.2` | `lg` | 8 columns   |           |
+
+> ![callout](https://user-images.githubusercontent.com/3793636/117873919-f6faba80-b265-11eb-81a5-039bdcd822e8.png)
+> See [[Image]], [[Video]], and [[Lightbox media viewer]] for more information.
+
+<!--    | `1.1.3` | `sm`        | 2 columns |     |
+| `1.1.4` | `xsm`   | 1 column    |           | -->
+
+<br />[Back to top](#wiki-wrapper)<br /><br /><br />
+
+### 2. Heading
+
+`optional` `string`
+
+An optional String of text that indicates the main point of the banner.
+
+> ![callout](https://user-images.githubusercontent.com/3793636/117873919-f6faba80-b265-11eb-81a5-039bdcd822e8.png)
+> See design specs <sup>[r1](#resources)</sup> for changes.
+
+<br />[Back to top](#wiki-wrapper)<br /><br /><br />
+
+### 3. Copy
+
+`optional` `markdown`
+
+An optional String of text formatted using [[Markdown utility]] with a limited scope.
+
+###### Only include
+
+> All other options should be ignored.
+
+|         | props             | description                                        |
+| :------ | :---------------- | :------------------------------------------------- |
+| `3.0.1` | `*italic*`        | The _italic_ option should be included.            |
+| `3.0.2` | `**bold**`        | The **bold** option should be included.            |
+| `3.0.3` | `<p>new line</p>` | The new line paragraph feature should be included. |
+
+> ![callout](https://user-images.githubusercontent.com/3793636/117873919-f6faba80-b265-11eb-81a5-039bdcd822e8.png)
+> See [[Markdown utility]] for more information.
+
+> ![callout](https://user-images.githubusercontent.com/3793636/117873919-f6faba80-b265-11eb-81a5-039bdcd822e8.png)
+> See design specs <sup>[r1](#resources)</sup> for changes.
+
+<br />[Back to top](#wiki-wrapper)<br /><br /><br />
+
+### 4. CTA
+
+`optional` `component`
+
+An optional [[CTA]] that helps navigate users to more information about this notice.
+
+###### Properties and values
+
+|         | props   | values          |
+| :------ | :------ | :-------------- |
+| `4.0.1` | `type`  | `local`         |
+| `4.0.2` | `style` | See logic below |
+
+###### Logic
+
+|         | condition           | description                        |
+| :------ | :------------------ | :--------------------------------- |
+| `4.0.3` | `@sm` `@md`         | `CTA` style is `text`              |
```

Comment:
There was a mistake in the spec. @sm should be the only breakpoint using CTA style text, and the whole banner is clickable. 

@md breakpoint should continue to use CTA button style.

Reply:
ok, I'll see about changing that